gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcCallContext.java

```diff
@@ -496,25 +543,39 @@ public CallOptions getCallOptions() {
     return callOptions;
   }
 
+  /** Backport of {@link #getStreamWaitTimeoutDuration()} */
```

Comment:
same comment as above - add this to the Javadoc as well

Reply:
Done